<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>基于Qt 和vlc的流媒体播放器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="QtPlayer.qrc">
    <normaloff>:/images/QtMedia.ico</normaloff>:/images/QtMedia.ico</iconset>
  </property>
  <property name="toolTip">
   <string>基于Qt 和vlc的流媒体播放器</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54);
</string>
  </property>
  <widget class="QWidget" name="widgetCentral">
   <layout class="QVBoxLayout" name="layoutCentral">
    <item>
     <widget class="QLabel" name="labFullscreen">
      <property name="toolTip">
       <string>全屏</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color:red</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>温馨提示：在视频区域双击鼠标以切换全屏和窗口模式╮(╯ɛ╰)╭</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frmScreen">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>视频</string>
      </property>
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <layout class="QGridLayout" name="layoutScreen">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QFrame" name="frmVideo">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>64</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/images/QtPlayer.jpg);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frmControl">
      <property name="toolTip">
       <string>控制</string>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QVBoxLayout" name="layoutControl">
       <item>
        <layout class="QHBoxLayout" name="layoutURL">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QLabel" name="labURL">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>URL</string>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(238, 238, 236);</string>
           </property>
           <property name="text">
            <string>URL:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editURL">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>URL</string>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(238, 238, 236);

</string>
           </property>
           <property name="cursorPosition">
            <number>0</number>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDirectoy">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(238, 238, 236);</string>
           </property>
           <property name="text">
            <string>路径</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutProgress">
         <item>
          <widget class="QSlider" name="sliderProgress">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>进度</string>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 170, 0);</string>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="pageStep">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labProgress">
           <property name="toolTip">
            <string>进度</string>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(211, 215, 207);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Panel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;00:00:00/00:00:00&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutButtons">
         <item>
          <widget class="QPushButton" name="btnPlay">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>播放</string>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 170, 0);</string>
           </property>
           <property name="text">
            <string>播放</string>
           </property>
           <property name="icon">
            <iconset resource="QtPlayer.qrc">
             <normaloff>:/images/Play.ico</normaloff>:/images/Play.ico</iconset>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPause">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>暂停</string>
           </property>
           <property name="text">
            <string>暂停</string>
           </property>
           <property name="icon">
            <iconset resource="QtPlayer.qrc">
             <normaloff>:/images/Pause.ico</normaloff>:/images/Pause.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnStop">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>停止</string>
           </property>
           <property name="text">
            <string>停止</string>
           </property>
           <property name="icon">
            <iconset resource="QtPlayer.qrc">
             <normaloff>:/images/Stop.ico</normaloff>:/images/Stop.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnFastBackward">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>快退</string>
           </property>
           <property name="text">
            <string>快退</string>
           </property>
           <property name="icon">
            <iconset resource="QtPlayer.qrc">
             <normaloff>:/images/FastBackward.ico</normaloff>:/images/FastBackward.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnFastForward">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>快进</string>
           </property>
           <property name="text">
            <string>快进</string>
           </property>
           <property name="icon">
            <iconset resource="QtPlayer.qrc">
             <normaloff>:/images/FastForward.ico</normaloff>:/images/FastForward.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacerVolume">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="labVolume">
           <property name="toolTip">
            <string>音量</string>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(238, 238, 236);</string>
           </property>
           <property name="text">
            <string>音量:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sliderVolume">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>64</width>
             <height>0</height>
            </size>
           </property>
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>音量</string>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>80</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
           <property name="tickInterval">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacerAbout">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnAbout">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string>关于</string>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(238, 238, 236);</string>
           </property>
           <property name="text">
            <string>关于...</string>
           </property>
           <property name="icon">
            <iconset resource="QtPlayer.qrc">
             <normaloff>:/images/About.ico</normaloff>:/images/About.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnQuit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string>退出</string>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(238, 238, 236);</string>
           </property>
           <property name="text">
            <string>退出</string>
           </property>
           <property name="icon">
            <iconset resource="QtPlayer.qrc">
             <normaloff>:/images/Quit.ico</normaloff>:/images/Quit.ico</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="QtPlayer.qrc"/>
 </resources>
 <connections/>
</ui>
